docs/content/troubleshooting.md

```diff
@@ -59,56 +59,74 @@ The most common causes of this exception are `bool` and `System.Boolean`, both o
 
 **Cause**<br/>
 
-This is a benign exception that is thrown when building a worker while burst compilation is turned on. Your worker was successfully built despite this error. This occurs becayse Unity's burst compiler doesn't yet fully support cross compilation (Windows to Linux, for example).
+This is a benign exception that is thrown when building a worker while burst compilation is turned on. Your worker was successfully built despite this error. This occurs because Unity's burst compiler doesn't yet fully support cross compilation (Windows to Linux, for example).
 
 **Workaround**<br/>
 
-To remove the error message, disable Burst compilation by unchecking `Jobs > Enable Burst Compilation` in the Unity Editor.
+To remove the error message, disable burst compilation by unchecking **Jobs** > **Enable Burst Compilation** in your Unity Editor.
 
+<br/>
 ## Warnings
+<br/>
+#### <li> <b>I get several warnings when opening the project for the first time</b>
 
-#### Several warnings when opening the project for the first time
-
+<br/>
 **Cause**<br/>
-These warnings originate from code written by Unity, `com.unity.mathematics` or `com.unity.entities` for example. These warnings are benign so you can ignore them.
+These warnings originate from code written by Unity, for example; `com.unity.mathematics` or `com.unity.entities`. 
+
+**Fix**<br/>
+These warnings are benign so you can ignore them.
 
-#### Several warnings when deleting a GameObject or ECS entity that represents a SpatialOS entity
+<br/>
+#### <li> <b> I get several warnings when deleting a GameObject or ECS entity that represents a SpatialOS entity</b>
 
 **Cause**<br/>
-These warnings occur when a [GameObject]({{urlRoot}}/content/glossary#gameobject) or [ECS entity]({{urlRoot}}/content/glossary#unity-ecs-entity) that represents a SpatialOS entity is deleted before its corresponding SpatialOS entity leaves the worker's view. The GDK is designed so that the GameObjects and ECS entities that represent SpatialOS entities should only be deleted after the SpatialOS entity they represent has left that worker's view. ECS entities are deleted automatically by the GDK when their corresponding SpatialOS entities leaves that worker's view. In the case of GameObjects, the SpatialOS Runtime sends a message to the Unity worker when a SpatialOS entity leaves that worker's view, the GDK passes this message onto you. It's your responsibility to act upon these commands in your code.
+These warnings occur when a [GameObject]({{urlRoot}}/content/glossary#gameobject) or [ECS entity]({{urlRoot}}/content/glossary#unity-ecs-entity) that represents a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity) is deleted before its corresponding SpatialOS entity leaves the [worker's view]({{urlRoot}}/content/glossary#worker-s-view). 
+
+The GDK expects a GameObject and ECS entity that represent a SpatialOS entity are only deleted after the SpatialOS entity they represent has left that worker's view: 
```

Comment:
I was going to change "ECS entity" to "ECS Entity", as I thought we started out by capitalising it in the GDK docs. However, looking in the GDK docs now, it's inconsistent. I seem to recall it was also inconsistent in Unity's basic ECS docs, in which case we can probably just make a call one way or the other.

Reply:
I think we ended up going with "entity" lower case. (A fair few of the Unity docs didn't get an editorial pass due to time panic so are likely to have inconsistencies - also I am not always on PRs.)